Formula/popeye.rb

```diff
@@ -0,0 +1,17 @@
+class Popeye < Formula
+  desc "Kubernetes cluster resource sanitizer"
+  homepage "https://popeyecli.io"
+  url "https://github.com/derailed/popeye/archive/refs/tags/v0.10.1.tar.gz"
+  sha256 "6dba28376f3016e49a597d1bb3b9365cdd5ba5cd6e21c848e1c97dca49d6bdaf"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    assert_match "#compdef _popeye popeye", shell_output("#{bin}/popeye completion zsh | head -n1")
```

Comment:
Can we try a basic functionality test, instead of trying to generate completions?

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Also, you don't need to pipe to `head` if you're doing `assert_match`.

Reply:
Thanks. I updated the test with some real world configuration.